pkg/util/template/func_test.go

```diff
@@ -27,4 +27,21 @@ func TestRFC3339(t *testing.T) {
 			So(RFC3339([]struct{}{}), ShouldEqual, "INVALID_DATE")
 		})
 	})
+
+	Convey("IsNil", t, func() {
+		Convey("it supports time.Time", func() {
```

Comment:
This Convey call can be removed. The label is irrelevant anyway.

Reply:
Removed 🙇🏻‍♀️